modules/gatsby/src/gatsby-node/create-webpack-config.js

```diff
@@ -1,5 +1,7 @@
   const {log, COLOR} = require('../utils/log');
 
+  const MODULE_NAME = '@jckr/ocular-gatsby';
```

Comment:
??? Change before landing?

Reply:
thanks for picking this up!!! yes this is to make it easier to test w/ local changes